internal/admission/validation/gateway/httproute.go

```diff
@@ -289,3 +289,30 @@ func validateWithKongGateway(
 func validationMsg(errMsgs []string) string {
 	return fmt.Sprintf("HTTPRoute failed schema validation: %s", strings.Join(errMsgs, ", "))
 }
+
+func validateHTTPRouteTimeoutBackendRequest(httproute *gatewayapi.HTTPRoute) error {
+	// TODO: remove the validate after we figure out how to handle granular timeout settings
+	// (allowing setting timeouts per rule and not enforcing the same timeout for every HTTPRoute's rule).
+	// https://github.com/Kong/kubernetes-ingress-controller/issues/5451
```

Comment:
Would you mind creating a placeholder issue for the "figure out granular timeout settings" part? That would be a prerequisite for #5451. It doesn't have to define the precise way how we can implement that, but the acceptance criteria are already known I think: "It's possible to set different timeout settings for every rule in a single HTTPRoute".

Reply:
Thank you. I have created the issue https://github.com/Kong/kubernetes-ingress-controller/issues/5478